Dockerfile

```diff
@@ -15,6 +15,7 @@ RUN set -ex \
 	&& apt-get update -qq \
 	&& apt-get install -qq -y --no-install-recommends \
 		gettext \
+		gnupg \
```

Comment:
I read somewhere in our code that installing `rng-tools` is useful to gain entropy while generating the keys, maybe we want to install that too.

Reply:
It's not listed here because it's a daemon that you'd typically run somewhere else. E.g. in our development environment we just advise to run it in the host and (the device is shared automatically with the container). In production environments I suspect you'd do similarly unless you want to run rng inside a container that can write to the device.